test/test_train_mp_imagenet.py

```diff
@@ -27,18 +27,6 @@
     '--test_only_at_end': {
         'action': 'store_true',
     },
-    # AMP only works with XLA:GPU
```

Comment:
Let's create a `test_train_mp_imagenet_amp.py` and only run the test if it is on GPU.